cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,62 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import HomePageObject from '../support/pages/home.pageObject';
+import SignUpPageObject from '../support/pages/signup.pageObject';
+import ArticlePageObject from '../support/pages/article.pageObject';
+
 describe('Article', () => {
+  let signUpPage;
+  let homePage;
+  let articlePage;
+
   before(() => {
+    cy.task('db:clear');
 
+    signUpPage = new SignUpPageObject();
+    homePage = new HomePageObject();
+    articlePage = new ArticlePageObject();
   });
 
   beforeEach(() => {
     cy.task('db:clear');
+    signUpPage.visit();
+    const user = signUpPage.registerUser();
```

Comment:
Verify that the `registerUser` method in `SignUpPageObject` returns an object with a `username` property. This is necessary for the `assertHeaderContainUsername` method to function correctly.